app/models/sequencescape_submission.rb

```diff
@@ -62,7 +64,31 @@ def assets=(asset_uuids)
   # @return [Array<String>] Array of asset uuids to submit
   #
   def assets
-    @asset_groups.pluck(:assets).flatten
+    @asset_groups.pluck(:assets)
+  end
+
+  def extra_barcodes_list
+    return nil unless extra_barcodes
+
+    extra_barcodes.split(' ')
+  end
```

Comment:
About the split - I'm wondering if we should be a bit more permissive in what we allow the separator to be? e.g. space, comma, new line? Like I think it does in LabWhere? 

Or else, we could put text above the box saying please  separate by one space?

Reply:
Yes, I'll fix that to support commas too.

Reply:
Ah I see you've got a test that checks it works for new line - didn't realise that would work!